WalletWasabi.Fluent/Views/Settings/BitcoinTabSettingsView.axaml

```diff
@@ -47,7 +47,7 @@
     </DockPanel>
 
     <StackPanel Spacing="10"
-                ToolTip.Tip="Coins under the dust threshold won't appear.">
+                ToolTip.Tip="Coins received from others to already used addresses won't appear below this amount.">
```

Comment:
I think the message should mention that this is an attack prevention thing. Because as of now it doesn't really tell me what is it for.

Reply:
done